release-notes/opensearch-release-notes-2.0.0-rc1.md

```diff
@@ -0,0 +1,606 @@
+# OpenSearch and Dashboards 2.0.0-rc1 Release Notes
+
+## Release Highlights
```

Comment:
Can we highlight the native KNN support and other notable Lucene 9.1 highlights ?

Reply:
lucene-based KNN isn't in 2.0. I believe it is being tracked here: https://github.com/opensearch-project/k-NN/issues/39

@vamshin pls confirm.